_episodes/01-what-is-git-CF.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
This file `_episodes/01-what-is-git-CF.md` looks like a copy of the episode 1 file that was used for editing. It should not be pushed to the main repository.